Core/src/main/java/com/plotsquared/core/configuration/Settings.java

```diff
@@ -411,12 +411,31 @@ public static boolean convertLegacy(File file) {
     }
 
 
+    @Deprecated
     @Comment("Schematic interface related settings")
     public static class Web {
         @Comment({"The web interface for schematics", " - All schematics are anonymous and private",
             " - Downloads can be deleted by the user",
             " - Supports plot uploads, downloads and saves",}) public static String URL =
             "https://schem.intellectualsites.com/plots/";
+        @Comment({"Whether or not the legacy web interface will be used for /plot download and /plot save",
+                 "Note that this will be removed in future versions. Updating to Arkitektonika is highly suggested"})
+        public static boolean LEGACY_WEBINTERFACE = false;
+    }
+
+    @Comment("Schematic web interface related settings")
+    public static class Arkitektonika {
+
+        @Comment("The url of the backend server (Arkitektonika)")
+        public static String BACKEND_URL = "https://ark.jacobandersen.dev/";
```

Comment:
It would be cool we could host Arkitektonika on Athion (@dordsor21 👀) so we have it is centralized alongside our other stuff and create an ALIAS Record so we can have it under `*.intellectualsites.com`

Reply:
I can host on the IS box, we just need to change the default before pushing v6.

Reply:
That would work as well. What do you mean by change the default? The existing web interface?

Reply:
I added a note for this on the project